requirements.txt

```diff
@@ -1,10 +1,3 @@
 # The order of packages is significant, because pip processes them in the order
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
```

Comment:
I removed all the requirements listed in `requirements.txt` file, because based on current state of the codebase, none of these are actually needed at this point. One can add requirement into this file at the same time when the code starts using it.

For `python setup.py install` to install requirements as well, those needs to be passed to the `install_requires` - which is missing right now and was already reported as https://github.com/Tendrl/alerting/issues/17. This issue #17 doesn't have any effect on this pull request, because as I already pointed out, no requirements are actually needed at this point.

So I think that fixing setup.py doesn't belong here, but into a pull request fixing issue #17 instead.

That said, I see your point, it would be better to limit are of interest of this pull request to tox and travis setup only - so I'm going to revert changes I did in `requirements.txt` file. Moreover it would make this pull request easier to merge.